packages/carbon-web-components/src/components/icon-button/icon-button.ts

```diff
@@ -0,0 +1,72 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2019, 2023
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { html } from 'lit';
+import { property, customElement } from 'lit/decorators.js';
+import { prefix } from '../../globals/settings';
+import '../tooltip/index';
+import '../button/index';
+import CDSButton from '../button/button';
+import { ICON_BUTTON_SIZE } from './defs';
+import styles from './icon-button.scss';
+
+export { ICON_BUTTON_SIZE };
+
+/**
+ * Icon Button
+ *
+ */
+@customElement(`${prefix}-icon-button`)
+class CDSIconButton extends CDSButton {
+  @property({ reflect: true, type: String })
+  align = 'top';
+
+  @property({ reflect: true, type: Boolean })
+  defaultOpen = false;
+
+  @property({ attribute: 'enter-delay-ms', type: Number })
+  enterDelayMs = 100;
+
+  @property({ attribute: 'exit-delay-ms', type: Number })
```

Comment:
```suggestion
  /**
   * Specify the duration in milliseconds to delay before hiding the tooltip
   */
  @property({ attribute: 'exit-delay-ms', type: Number })
```

In `@carbon/react` this attr is called `leaveDelayMs`, should we change to match?

Reply:
updated to be leaveDelayMs!